Clean up #ifndef/#defines
diff -urN usr/local/include/config.h usr/local/include/config.h
--- usr/local/include/config.h	2019-06-26 09:18:18.000000000 +0100
+++ usr/local/include/config.h	2019-06-29 00:12:19.065391458 +0100
@@ -1,5 +1,5 @@
-#ifndef __CONFIG_H__
-#define __CONFIG_H__
+#ifndef __QHYCCDCONFIG_H__
+#define __QHYCCDCONFIG_H__
 
 
 
diff -urN usr/local/include/qhyccd.h usr/local/include/qhyccd.h
--- usr/local/include/qhyccd.h	2019-06-28 23:29:37.443015088 +0100
+++ usr/local/include/qhyccd.h	2019-06-29 00:12:19.874361269 +0100
@@ -1,3 +1,5 @@
+#ifndef __QHYCCD_H__
+#define __QHYCCD_H__
 
 #include <qhyccd/qhyccderr.h>
 #include <qhyccd/qhyccdcamdef.h>
@@ -14,9 +16,6 @@
 #include "cyapi.h"
 #endif
 
-#ifndef __QHYCCD_H__
-#define __QHYCCD_H__
-
 typedef void qhyccd_handle;
 
 
diff -urN usr/local/include/qhyccdstruct.h usr/local/include/qhyccdstruct.h
--- usr/local/include/qhyccdstruct.h	2019-06-28 23:29:39.256948075 +0100
+++ usr/local/include/qhyccdstruct.h	2019-06-29 00:12:20.753328379 +0100
@@ -1,14 +1,8 @@
-
-#include <qhyccd/config.h>
-
-
-
-
-
-
 #ifndef __QHYCCDSTRUCTDEF_H__
 #define __QHYCCDSTRUCTDEF_H__
 
+#include <qhyccd/config.h>
+
 #if defined (_WIN32)
 #ifndef EXPORTFUNC
 #define EXPORTFUNC extern "C" __declspec(dllexport)
