Allow compilation under C
diff -urN usr/local/include/qhyccd.h usr/local/include/qhyccd.h
--- usr/local/include/qhyccd.h	2019-06-29 00:12:19.874361269 +0100
+++ usr/local/include/qhyccd.h	2019-06-29 00:13:25.178919468 +0100
@@ -6,8 +6,10 @@
 #include <qhyccd/qhyccdstruct.h>
 #include <stdint.h>
 #include <qhyccd/config.h>
+#if defined(_WIN32) || defined(__cplusplus)
 #include <functional>
 #include <string>
+#endif
 
 
 
@@ -25,8 +27,11 @@
 EXPORTC void STDCALL SetQHYCCDLogLevel(uint8_t logLevel);
 
 #if (defined(__linux__ )&&!defined (__ANDROID__)) ||(defined (__APPLE__)&&defined( __MACH__)) ||(defined(__linux__ )&&defined (__ANDROID__))
-
+#ifdef __cplusplus
 EXPORTC void STDCALL SetQHYCCDLogFunction(std::function<void(const std::string &message)> logFunction);
+#else
+EXPORTC void STDCALL SetQHYCCDLogFunction(void (*) ( const char* ));
+#endif
 EXPORTC void STDCALL SetQHYCCDBufferNumber(uint32_t BufNumber);
 
 #endif
XXXX
diff -urN usr/local/include/qhyccdstruct.h usr/local/include/qhyccdstruct.h
--- usr/local/include/qhyccdstruct.h	2019-06-29 00:12:20.753328379 +0100
+++ usr/local/include/qhyccdstruct.h	2019-06-29 00:13:25.881893455 +0100
@@ -14,9 +14,15 @@
 #define EXPORTC extern "C"
 #endif
 #else
+#ifdef __cplusplus
 #define EXPORTFUNC extern "C"
 #define STDCALL
 #define EXPORTC extern "C"
+#else
+#define EXPORTFUNC
+#define STDCALL
+#define EXPORTC
+#endif
 #endif
 
 #include <stdint.h>
@@ -100,7 +106,7 @@
 }
 CCDREG;
 
-struct BIOREG
+typedef struct BIOREG
 {
   uint16_t LineSize;
   uint16_t PatchNumber;
@@ -112,7 +118,7 @@
   uint8_t  BIOCCD_Mode;
   uint8_t  BIOCCD_Video;
   uint8_t  SDRAM_Bypass;
-};
+} BIOREG;
 
 
 
@@ -153,7 +159,7 @@
  *
  * List of the function could be control
  */
-enum CONTROL_ID
+typedef enum CONTROL_ID
 {
 /*0*/  CONTROL_BRIGHTNESS = 0, //!< image brightness
 /*1*/  CONTROL_CONTRAST,       //!< image contrast
@@ -229,23 +235,23 @@
 
 /* Do not Put Item after  CONTROL_MAX_ID !! This should be the max index of the list */
 /*Last One */  CONTROL_MAX_ID
-};
+} CONTROL_ID;
 
 /**
  * debayer mode for mono to color */
-enum BAYER_ID
+typedef enum BAYER_ID
 {
   BAYER_GB = 1,
   BAYER_GR,
   BAYER_BG,
   BAYER_RG
-};
+} BAYER_ID;
 
-enum CodecID
+typedef enum CodecID
 {
   NONE_CODEC,
   H261_CODEC
-};
+} CodecID;
 
 
 
