Allow compilation under C
diff -urN usr/local/include/qhyccd.h usr/local/include/qhyccd.h
--- usr/local/include/qhyccd.h	2023-12-27 17:22:52.945878211 +0000
+++ usr/local/include/qhyccd.h	2023-12-27 17:24:19.296664078 +0000
@@ -6,8 +6,10 @@
 #include <qhyccd/qhyccdstruct.h>
 #include <stdint.h>
 #include <qhyccd/config.h>
+#if defined(_WIN32) || defined(__cplusplus)
 #include <functional>
 #include <string>
+#endif
 
 
 
@@ -26,8 +28,11 @@
 EXPORTC void STDCALL SetQHYCCDLogLevel(uint8_t logLevel);
 
 #if (defined(__linux__ )&&!defined (__ANDROID__)) ||(defined (__APPLE__)&&defined( __MACH__)) ||(defined(__linux__ )&&defined (__ANDROID__))
-
+#ifdef __cplusplus
 EXPORTC void STDCALL SetQHYCCDLogFunction(std::function<void(const std::string &message)> logFunction);
+#else
+EXPORTC void STDCALL SetQHYCCDLogFunction(void (*) ( const char* ));
+#endif
 EXPORTC void STDCALL SetQHYCCDBufferNumber(uint32_t BufNumber);
 
 #endif
@@ -935,12 +940,20 @@
 EXPORTFUNC void STDCALL QHYCCDTestFlashULVOError(qhyccd_handle *handle);
 EXPORTFUNC void STDCALL QHYCCDSetFlashInitPWM(qhyccd_handle *handle,uint8_t pwm);
 EXPORTFUNC void STDCALL QHYCCDGetDebugDataD3(qhyccd_handle *handle, char* debugData_raw64);
+#ifdef __cplusplus
 EXPORTFUNC uint32_t STDCALL QHYCCDSolve(int timeout_s, float scale_l, float  scale_h,float center_ra, float center_dec,float center_r, float& s_ra, float& s_dec,float& s_size_x,float& s_size_y, float& s_rotation);
+#else
+EXPORTFUNC uint32_t STDCALL QHYCCDSolve(int timeout_s, float scale_l, float  scale_h,float center_ra, float center_dec,float center_r, float* s_ra, float* s_dec,float* s_size_x,float* s_size_y, float* s_rotation);
+#endif
 EXPORTFUNC void STDCALL QHYCCDEqualizeHistogram(uint8_t * pdata, int width, int height, int bpp);
 void  QHYCCDGetDebugControlID(CONTROL_ID controlId, bool hasValue, bool isSetValue, double value);
 
 
+#ifdef __cplusplus
 EXPORTFUNC int STDCALL QHYCCD_fpga_list(struct fpga_info_list &list);
+#else
+EXPORTFUNC int STDCALL QHYCCD_fpga_list(void *list);
+#endif
 EXPORTFUNC uint32_t STDCALL QHYCCD_fpga_open(int id);
 EXPORTFUNC void STDCALL QHYCCD_fpga_close();
 EXPORTFUNC int STDCALL QHYCCD_fpga_send(int chnl, void * data, int len, int destoff, int last, uint64_t timeout);
@@ -1038,4 +1051,4 @@
 EXPORTFUNC uint32_t STDCALL QHYCCD_GainValueToDbGain(qhyccd_handle *h,double gainvalue,double *dbgain);
 EXPORTFUNC uint32_t STDCALL QHYCCD_curveSystemGain(qhyccd_handle *handle,double gainV,double *systemgain);
 EXPORTFUNC uint32_t STDCALL QHYCCD_curveFullWell(qhyccd_handle *handle,double gainV,double *fullwell);
-EXPORTFUNC uint32_t STDCALL QHYCCD_curveReadoutNoise(qhyccd_handle *handle,double gainV,double *readoutnoise);
\ No newline at end of file
+EXPORTFUNC uint32_t STDCALL QHYCCD_curveReadoutNoise(qhyccd_handle *handle,double gainV,double *readoutnoise);
diff -urN usr/local/include/qhyccdstruct.h usr/local/include/qhyccdstruct.h
--- usr/local/include/qhyccdstruct.h	2023-12-27 16:33:42.925634718 +0000
+++ usr/local/include/qhyccdstruct.h	2023-12-27 16:55:31.549953721 +0000
@@ -14,9 +14,15 @@
 #define EXPORTC extern "C"
 #endif
 #else
+#ifdef __cplusplus
 #define EXPORTFUNC extern "C"
 #define STDCALL
 #define EXPORTC extern "C"
+#else
+#define EXPORTFUNC
+#define STDCALL
+#define EXPORTC
+#endif
 #endif
 
 #include <stdint.h>
@@ -100,7 +106,7 @@
 }
 CCDREG;
 
-struct BIOREG
+typedef struct BIOREG
 {
   uint16_t LineSize;
   uint16_t PatchNumber;
@@ -112,7 +118,7 @@
   uint8_t  BIOCCD_Mode;
   uint8_t  BIOCCD_Video;
   uint8_t  SDRAM_Bypass;
-};
+} BIOREG;
 
 
 
@@ -153,7 +159,7 @@
  *
  * List of the function could be control
  */
-enum CONTROL_ID
+typedef enum CONTROL_ID
 {
 /*0*/  CONTROL_BRIGHTNESS = 0, //!< image brightness
 /*1*/  CONTROL_CONTRAST,       //!< image contrast
@@ -261,25 +267,25 @@
 /*1029*/ CONTROL_GAINdB,			//!<uesed to test dBGain control  //CONTROL_dB_TO_GAIN
 /*1030*/ CONTROL_DPC,            //!<Turn on or off the image DPC function(Remove thermal noise)
 /*1031*/ CONTROL_DPC_value       //!<value the image DPC function(Remove thermal noise)
-};
+} CONTROL_ID;
 
 /**
  * debayer mode for mono to color */
-enum BAYER_ID
+typedef enum BAYER_ID
 {
   BAYER_GB = 1,
   BAYER_GR,
   BAYER_BG,
   BAYER_RG
-};
+} BAYER_ID;
 
-enum CodecID
+typedef enum CodecID
 {
   NONE_CODEC,
   H261_CODEC
-};
+} CodecID;
 
-enum ImgProc
+typedef enum ImgProc
 {
   NOPROC = 0,
   ROTATION180,
@@ -287,7 +293,7 @@
   ROTATION90R,
   MIRRORH,
   MIRRORV
-};
+} ImgProc;
 
 
 
