Allow compilation under C
diff -urN usr/local/include/qhyccd.h usr/local/include/qhyccd.h
--- usr/local/include/qhyccd.h	2019-06-29 00:12:19.874361269 +0100
+++ usr/local/include/qhyccd.h	2019-06-29 00:13:25.178919468 +0100
@@ -6,11 +6,9 @@
 #include <qhyccd/qhyccdstruct.h>
 #include <stdint.h>
 #include <qhyccd/config.h>
+#if defined(_WIN32) || defined(__cplusplus)
 #include <functional>
-
-
-
-
+#endif
 
 #if defined (_WIN32)
 #include "cyapi.h"
@@ -27,8 +25,11 @@
 EXPORTC void STDCALL SetQHYCCDLogLevel(uint8_t logLevel);
 
 #if defined(__linux__ )&&!defined (__ANDROID__)
-
+#ifdef __cplusplus
 EXPORTC void STDCALL SetQHYCCDLogFunction(std::function<void(const std::string &message)> logFunction);
+#else
+EXPORTC void STDCALL SetQHYCCDLogFunction(void (*) ( const char* ));
+#endif
 EXPORTC void STDCALL SetQHYCCDBufferNumber(uint32_t BufNumber);
 
 #endif
diff -urN usr/local/include/qhyccdstruct.h usr/local/include/qhyccdstruct.h
--- usr/local/include/qhyccdstruct.h	2019-06-29 00:12:20.753328379 +0100
+++ usr/local/include/qhyccdstruct.h	2019-06-29 00:13:25.881893455 +0100
@@ -14,9 +14,15 @@
 #define EXPORTC extern "C"
 #endif
 #else
+#ifdef __cplusplus
 #define EXPORTFUNC extern "C"
 #define STDCALL
 #define EXPORTC extern "C"
+#else
+#define EXPORTFUNC
+#define STDCALL
+#define EXPORTC
+#endif
 #endif
 
 #include <stdint.h>
@@ -94,7 +100,7 @@
 }
 CCDREG;
 
-struct BIOREG
+typedef struct BIOREG
 {
   uint16_t LineSize;
   uint16_t PatchNumber;
@@ -106,7 +112,7 @@
   uint8_t  BIOCCD_Mode;
   uint8_t  BIOCCD_Video;
   uint8_t  SDRAM_Bypass;
-};
+} BIOREG;
 
 
 
@@ -147,7 +153,7 @@
  *
  * List of the function could be control
  */
-enum CONTROL_ID
+typedef enum CONTROL_ID
 {
   CONTROL_BRIGHTNESS = 0, //!< image brightness
   CONTROL_CONTRAST,       //!< image contrast
@@ -230,23 +236,23 @@
   CONTROL_MAX_ID,
   CAM_HUMIDITY			//!<check if camera has	 humidity sensor  20191021 LYL Unified humidity function
 
-};
+} CONTROL_ID;
 
 /**
  * debayer mode for mono to color */
-enum BAYER_ID
+typedef enum BAYER_ID
 {
   BAYER_GB = 1,
   BAYER_GR,
   BAYER_BG,
   BAYER_RG
-};
+} BAYER_ID;
 
-enum CodecID
+typedef enum CodecID
 {
   NONE_CODEC,
   H261_CODEC
-};
+} CodecID;
 
 typedef struct _QHYCamReadModeInfo
 {
