Clean up #include and #ifdef statements
--- usr/local/include/qhyccdstruct.h	2019-02-14 22:51:51.425255939 +0000
+++ usr/local/include/qhyccdstruct.h	2019-02-14 22:52:26.754638771 +0000
@@ -1,10 +1,10 @@
+#ifndef __QHYCCDSTRUCTDEF_H__
+#define __QHYCCDSTRUCTDEF_H__
+
 #if defined (_WIN32)
 #include <windows.h>
 #endif
 
-#ifndef __QHYCCDSTRUCTDEF_H__
-#define __QHYCCDSTRUCTDEF_H__
-
 #if defined (_WIN32)
 #ifndef EXPORTFUNC
 #define EXPORTFUNC extern "C" __declspec(dllexport)
@@ -21,8 +21,6 @@
 #define EXPORTC extern "C"
 #endif
 
-#include "stdint.h"
-
 /**
  * usb vendor request command
  */
--- usr/local/include/qhyccd.h	2019-02-14 22:55:55.630626603 +0000
+++ usr/local/include/qhyccd.h	2019-02-14 22:52:05.346638086 +0000
@@ -1,17 +1,16 @@
+#ifndef __QHYCCD_H__
+#define __QHYCCD_H__
 
-#include "qhyccderr.h"
-#include "qhyccdcamdef.h"
-#include "qhyccdstruct.h"
-#include "stdint.h"
+#include <stdint.h>
+#include <qhyccderr.h>
+#include <qhyccdcamdef.h>
+#include <qhyccdstruct.h>
 #include <functional>
 
 #if defined (_WIN32)
 #include "cyapi.h"
 #endif
 
-#ifndef __QHYCCD_H__
-#define __QHYCCD_H__
-
 #if defined (_WIN32)
 typedef CCyUSBDevice qhyccd_handle;
 #endif
